pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', 'build/x86_64_xenial' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'run', '-v', '/workspace:/workspace', '-w', '/workspace/src/',  'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', '/bin/bash', '-c', 'make purge && make -j4 DISTRO=xenial deb' ]
+- name: 'ubuntu:xenial'
+  args: [ 'cp', '/workspace/src/stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb', '/workspace/pkg/docker/stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '--build-arg', 'package=stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial', '-f', 'pkg/docker/Dockerfile-xenial', 'pkg/docker' ]
+images:
+- 'gcr.io/$PROJECT_ID/metadata-agent-xenial'
+substitutions:
+  _METADATA_AGENT_VERSION: 0.0.21-1
```

Comment:
Any reason we can't use the default version in the `Makefile`? Moreover, the `Dockerfile` doesn't actually care about the version (note that it's copied into the image as just a plain `stackdriver-metadata.deb`). So can we not just build it under that unversioned name?